app/services/custom_landing_page/landing_page.rb.template

```diff
@@ -0,0 +1,304 @@
+module CustomLandingPage
```

Comment:
This duplicates `app/services/custom_landing_page/example_data.rb`. can you reuse?

Reply:
Could you give suggestions on how I can reuse here? I duplicated some but not all of `ExampleData` and the module name differs by a bit.

Would a copy + regex substitution suffice?

Reply:
1) Stick to `example_data.rb` file. 
2) You can keep your version of `DATA_STR`, just set the `copyright` text back to "This website is powered by Sharetribe marketplace platform."
3) in the rake task copy the `example_data.rb` and replace the module name to `StaticData` in the new file
4) remove the landing_page.rb.template